speccpu2017/run_speccpu

```diff
@@ -28,6 +28,15 @@ arguments="$@"
 run_status="Ran"
 gen_args_back="$@"
 disks_found=0
+installed=0
+speccpu_run_dir=""
+speccpu_run="/speccpu_run"
+speccpu_iso_mnt="/speccpu"
+speccpu_iso_file=""
+uploads=""
+give_usage=0
+use_mfs=""
```

Comment:
For the option let's call it "no_disk" which we document as "use the existing filesystem with the most amount of free space".